lib/virtfs/ext3.rb

```diff
@@ -1 +1,16 @@
 require "virtfs/ext3/version"
+require_relative 'ext3/alloc_bitmap'
```

Comment:
See @Fryguy comment on my PR https://github.com/ManageIQ/virtfs-xfs/pull/1 - "Minor, but it's common in gems to just refer to file assuming lib is on the path. That is, instead of require_relative 'xfs/whatever', prefer require 'virtfs/xfs/whatever'".  If this is desired we should be consistent throughout all the VirtFS and VirtFS-fstype gems.